src/templates/dataGridTable.jsx

```diff
@@ -26,6 +37,16 @@ const propTypes = {
         'small',
         'medium',
     ]),
+    sortable: PropTypes.arrayOf(
+        PropTypes.shape({
+            disabled: PropTypes.bool,
+            filter: PropTypes.string,
+            group: PropTypes.string,
+            handle: PropTypes.bool,
+            onChange: PropTypes.func,
+            sort: PropTypes.bool,
+        }),
```

Comment:
Should we consider a constant for this Prop Type Shape?
_Not a Blocker_

Reply:
Fixed.